source/beamline/BioXAS/BioXASZebraTimeSeconds.cpp

```diff
@@ -0,0 +1,222 @@
+#include "BioXASZebraTimeSeconds.h"
+#include "actions3/AMActionSupport.h"
+#include "actions3/AMListAction3.h"
+
+BioXASZebraTimeSeconds::BioXASZebraTimeSeconds(const QString &name, QObject *parent) :
```

Comment:
I know this is really pedantic, but I feel like the name should include Control in some capacity since it is mapping a concept to PVs.  Even if it was just BioXASZebraTimeSecondsControl... I don't know.  Thoughts @acquaman/acquaman-working-group?